frontend/src/css/custom.css

```diff
@@ -31,7 +31,7 @@ html[data-theme="dark"] .docusaurus-highlight-code-line {
   color-scheme: initial;
 }
 
-input[type="text"] {
+input[type="text"], :not(#fakeID#fakeId#fakeID) select {
```

Comment:
This is not great CSS. Every select box has a couple of classes attached to it with long auto-generated names that I don't really want to use directly. They seem like they may not be reliable over time.
In order to make a more specific selector, I've used this hack https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#increasing_specificity_by_duplicating_selector

I might have a look at opening a PR to implement this and https://github.com/badges/shields/pull/9324 upstream at the component layer so we clean this up a bit.